README.md

```diff
@@ -98,42 +88,341 @@ See [variables.tf] and [examples/] for details and use-cases.
 
 #### Main Resource Configuration
 
+##### VPC
+
+- **`vpc_name`**: *(Required `string`)*
+
+  The Name of the VPC. This will be used to tag resources with Names by default.
+  Default is `"main"`.
+
+- **`cidr_block`**: *(Required `string`)*
+
+  The CIDR block for the VPC.
```

Comment:
```suggestion
  The primary CIDR block for the VPC.
```

might be better to use primary as we will introduce secondary later down the road

Reply:
we can update this wording then. atm it would just confuse as there is no secondary.... and secondary is a rarely used feature for most scales.. ;)